imod/mf6/package.py

```diff
@@ -533,17 +533,37 @@ def mask(self, domain: GridDataArray) -> Any:
         masked: Package
             The package with part masked.
         """
+
+        if any([d not in ["x", "y", "layer", "mesh2d_nFaces", "dx", "dy"] for d in domain.coords]):
```

Comment:
Same comment as before: Can we use a better name then d?

Reply:
done